src/config.rs

```diff
@@ -68,6 +74,98 @@ impl Config {
     }
 }
 
+#[cfg(unix)]
+type Child = std::process::Child;
+#[cfg(windows)]
+type Child = create_process_w::Child;
+
+#[cfg(unix)]
+#[derive(Serialize, Deserialize)]
+pub struct SubProcess {
+    pub command: String,
+    pub foreground: bool,
+    #[serde(default)]
+    pub keep_on_exit: bool,
+    pub working_dir: Option<PathBuf>,
+    pub stdout: Option<bool>,
+    pub stderr: Option<bool>,
+}
+#[cfg(windows)]
+#[derive(Serialize, Deserialize)]
+pub struct SubProcess {
+    pub command: String,
+    pub foreground: bool,
+    #[serde(default)]
+    pub keep_on_exit: bool,
+    pub working_dir: Option<PathBuf>,
+    pub inherit_handles: Option<bool>,
+}
```

Comment:
You can scope fields to a specific target:

```suggestion
#[derive(Serialize, Deserialize)]
pub struct SubProcess {
    pub command: String,
    pub foreground: bool,
    #[serde(default)]
    pub keep_on_exit: bool,
    pub working_dir: Option<PathBuf>,
    #[cfg(unix)]
    pub stdout: Option<bool>,
    #[cfg(unix)]
    pub stderr: Option<bool>,
    #[cfg(windows)]
    pub inherit_handles: Option<bool>,
}
```

Reply:
TIL, so cool